### PR TITLE
Adjust basic model to FW's suggested pattern

### PR DESCRIPTION
**Pull request by fwigorcferreira**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -165,16 +164,16 @@ func swap<Thing>(_ a: inout Thing, _ b: inout Thing)
 
 ### Language
 
-Use US English spelling to match Apple's API.
+Use UK English spelling to match Apple's API.
```
Comment (ghost) [inline README.markdown]: I believe this is not correct, must confirm with Fabio, but I do believe spelling should be US to be consistent with international and Apple coding de-facto "standards".

File: README.markdown [hunk 2]
```diff
@@ -279,12 +280,12 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, dictionaries `["A" : 1.2]`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```
Comment (ghost) [inline README.markdown]: I believe dictionaries still follow `["key": "value"]` with no space on the left, and one space on the right. It's only the empty dictionary `[:]` that carries no spaces around

Comment (ghost) [reply README.markdown]: [http://ericasadun.com/2016/03/25/swift-colon-care-high-fiber-usage/](url)

**Review by ghost — DISMISSED** [review]

File: README.markdown (hunk 1 above)
Comment (fwigorcferreira) [reply README.markdown]: 👍

File: README.markdown (hunk 2 above)
Comment (fwigorcferreira) [reply README.markdown]: :+1:

File: README.markdown [hunk 3]
```diff
@@ -279,12 +280,12 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, dictionary type declaration `[String : CGFloat]`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```
Comment (fwigorcferreira) [inline README.markdown:283]: @FWPedro I removed the wrong dictionary explanation, and specified the rule exception for the dictionary type

**Review by ghost — APPROVED** [review]

**Pull request merged**